apps/backend/README.md

```diff
@@ -6,6 +6,8 @@ To view the Bootify.io Setup, visit [this link](https://bootify.io/app/1AFNHOA9N
 
 ## Development
 
+### Prerequisites
+
 During development it is recommended to use the profile `development`. In IntelliJ `--spring.config.additional-location=classpath:application-development.yml --spring.profiles.active=development` can be
 added in the Run Configuration after enabling this property in "Modify options". Create your own
 `application-development.yml` file to override settings for development.
```

Comment:
I ha no dr folder Pfad ergänzt gha: Create your own `application-development.yml` in file in the `apps\backend\src\main\resources\` directory to override settings for development